### PR TITLE
Create insertJob function

### PR DESCRIPTION
**Pull request by shravan1k**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Made a function to insert a job into the database.

### Testing
How did you confirm your changes worked? 
- Ran the server with the cron job making calls to the function. I then looked at the database to see if it was updated with the inserted entries, which it was.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

image

image

| [TODO Screenshot Title] |
| --- |
| [TODO Screenshot Link] |

### CODE REVIEW

**Review by eliasfang — APPROVED** [review]
Super cool that you got the table working. LGTM!

File: app/database/dbConfig.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const knex = require('knex');
 
-const db = process.eng.DB_ENV || 'development';
+const db = process.env.DB_ENV || 'development';
```
Comment (mingkaili) [inline app/database/dbConfig.js:3]: Good job catching this typo

**Review by mingkaili — APPROVED** [review]

File: app/models/jobsModel.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const db = require('../database/dbConfig.js')
+
+async function insertJob(job){
```
Comment (mingkaili) [inline app/models/jobsModel.js:11]: It's also good to write a couple lines of comment as javadoc comment just to define the parameter and how the json shoule look like

**Review by mingkaili — CHANGES_REQUESTED** [review]
Great start, and you have a lot of good stuff in here, leave some comments on the function just to explain the parameter type and the json format

File: app/models/jobsModel.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const db = require('../database/dbConfig.js')
+
+/**
+ * Inserts a job into the database.
+ * scheduled_time is an optional field.
+ * 
+ * @param {"class_id" : integer, 
+ *         "scheduled_time" : DateTime,
```
Comment (mingkaili) [inline app/models/jobsModel.js]: DateTime is actually not a type in js, it is more clear to just change this to Date, a Date object in js contains date and time

**Review by mingkaili — APPROVED** [review]

**Review by jacobayu — APPROVED** [review]

**Pull request merged**